doc/code.html

```diff
@@ -414,14 +417,14 @@ <h3 id="Library">Your first library</h3>
 </p>
 
 <pre>
-bin/
-    hello                 # command executable
-src/
-    github.com/user/
-        hello/
-            hello.go      # command source
-        stringutil/
-            reverse.go    # package source
+$GOPATH/bin/
+		hello                 # command executable
+		
+~/myproject/
+		hello/
+				hello.go      # command source
+		stringutil/
```

Comment:
This should now be placed inside of the `hello` folder, yes? (and remember the go.mod ;) )